app/controllers/DocumentController.js

```diff
@@ -1,23 +1,21 @@
 import models from '../models/dbconnect';
-import Helpers from './helperMethods';
+import Helpers from './Helpers';
 
 const helperMethods = new Helpers();
 
 /**
-  * Class DocControl. Handles document functions
-  *
-  * @returns {void}
-  */
-export default class DocControl {
+ * Class DocControl. Handles document functions
```

Comment:
###Nitpick

This class name in the doc block does not match the actual class name.